argus/backends/azure/azure_backend.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2017 Cloudbase Solutions Srl
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import abc
+import six
+
+from argus.backends import base
+from argus.backends import windows
+from argus.backends.azure import client as azure_client
+
+from argus import config as argus_config
+from argus import log as argus_log
+
+CONFIG = argus_config.CONFIG
+
+RETRY_COUNT = 50

```

Comment:
Wouldn't it be better to have these as configurable options in default? Also I don't find these variables being used anywhere